pkgs/tools/package-management/nix-update/default.nix

```diff
@@ -1,21 +1,21 @@
 { lib
-, buildPythonApplication
+, python3
 , fetchFromGitHub
 , nix
```

Comment:
this update adds a nix-prefetch-git as a runtime dependency

```suggestion
, nix
, nix-prefetch-git
```

Reply:
Thanks, fixed.